flintrock/core.py

```diff
@@ -482,6 +483,58 @@ def _run_asynchronously(*, partial_func: functools.partial, hosts: list):
         loop.close()
 
 
+def get_java_major_version(client: paramiko.client.SSHClient):
+    possible_cmds = [
+        "$JAVA_HOME/bin/java -version",
+        "java -version"
+    ]
+
+    for command in possible_cmds:
+        try:
+            output = ssh_check_output(
```

Comment:
Actually, my main test case was Amazon Linux where Java 1.7 is installed by default. I've confirmed the function returning (1, 7).

But just in case, I'll add `2>&1` to the code.